programs/survival/carried_totem.sc

```diff
@@ -1,15 +1,31 @@
 // Allow a Totem of Undying anywhere in the inventory to activate
 
+__config() -> (
+   m(
+      l('stay_loaded','true')
+   )
+);
+
 __on_player_takes_damage(player, amount, source, source_entity) -> (
 	totem = 'totem_of_undying';
 	if(player~'health' <= amount
 	&& source != 'outOfWorld'
 	&& player~'holds':0 != totem
 	&& query(player, 'holds', 'offhand'):0 != totem
-	&& inventory_remove(player, totem),
-		// borrow the offhand slot
+	&& inventory_find(player, totem),
```

Comment:
This will evaluate to false if the totem is in the first slot. I think you want `inventory_find(player, totem) != null`